drops/azure_edge_extensions_lnm_mq_isa95.json

```diff
@@ -0,0 +1,30 @@
+{
+    "Title": "Azure Edge Extensions AIO ISA-95 LNM and MQ",
```

Comment:
Maybe try to define the goal here -> Is this an automated delpoyment?

Reply:
done and also avoided to use the acronyms, except for MQ for is the commonly used wording for "Message Queueing"